deploy-manage/deploy/cloud-enterprise/change-endpoint-urls.md

```diff
@@ -1,4 +1,7 @@
 ---
+applies_to:
+  deployment:
+    ece: all
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-administering-endpoints.html
 ---
```

Comment:
this page has a weird admonition on it

<img width="746" alt="image" src="https://github.com/user-attachments/assets/da33d2b7-0056-439d-a11e-b738a7129199" />